src/snapred/backend/recipe/algorithm/RawVanadiumCorrectionAlgorithm.py

```diff
@@ -29,10 +28,6 @@ def PyInit(self):
             MatrixWorkspaceProperty("BackgroundWorkspace", "", Direction.Input, PropertyMode.Mandatory),
```

Comment:
I'm wondering if "InputWorkspace" should be changed to "VanadiumWorkspace"?

Reply:
I am not sure. Yes, it would make the property name more meaningful. But on the other hand,  `InputWorkspace`/`OutputWorkspace` is a name pattern used in many `Mantid` algorithms. Also, to be consistent, if we went with `VanadiumWorkspace`, shouldn't we then change `OutputWorkspace` to `CorrectedVanadiumWorkspace`?